5.optimization/scripts/1.boxplots.r

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env r
+# coding: utf-8
+
+# In[1]:
+
+
+# Load necessary libraries
+library(ggplot2)
+library(dplyr)
+library(arrow)
+
+
+# In[2]:
+
+
+# Set the file path to your results folder
+file_dir <- file.path("results")
```

Comment:
Consider creating the folder at the beginning of the script. Just in case it does not exist and no errors get raised. 

```suggestion
file_dir <- file.path("results")
dir.create(file_dir, showWarnings = FALSE) 
```

Reply:
done!